modules/swagger-codegen/src/main/resources/ruby/partial_model_generic.mustache

```diff
@@ -58,6 +58,11 @@
 
       # convert string to symbol for hash key
       attributes = attributes.each_with_object({}) { |(k, v), h| h[k.to_sym] = v }
+      {{#parent}}
+
+      # call parent's initialize
+      super(attributes)
+      {{/parent}}
```

Comment:
The parent constructor is called with the values passed in to the constructor of the child